lib/invoice_repository.rb

```diff
@@ -117,4 +117,21 @@ def find_customer(id)
   def find_merchant(id)
     sales_engine.find_merchant_by_id(id)
   end
+
+  def create(inputs)
+    line = {
+      id:          "#{invoices.last.id + 1}",
+      customer_id: inputs[:customer].id,
+      merchant:    inputs[:merchant].id,
+      status:      inputs[:status],
+      created_at:  "#{Date.new}",
+      updated_at:  "#{Date.new}"
+      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.